app/models/hmis/build_report.rb

```diff
@@ -0,0 +1,66 @@
+module Hmis
+  class BuildReport
+    def run!
+      # we just flush the whole damned thing every time
+      Hmis::Report.delete_all
```

Comment:
Checkout the Utility class in the warehouse, let's implement the truncate method, it will be way faster for deletes.